docs/components.md

```diff
@@ -67,7 +67,78 @@ Card for use in card lists
             </span>
         </flag>
     </section>
-    <footer class="small end block-component">CurrentStatus</footer>
+    <footer class="small block-component status">Unknown/No Status</footer>
+</div>
+{% endsnippet %}
+
+
+{% snippet html %}
+<div class="card">
+    <section class="flex-row">
+        <section class="flex-column">
+            <div class="item-title block-component">
+                <span class="item-number">1.2.3.4.5</span>
+                <span class="name">Some Thing</span>
+            </div>
+        </section>
+    </section>
+    <footer class="small block-component status notstarted">Not Started</footer>
+</div>
+{% endsnippet %}
+
+{% snippet html %}
+<div class="card">
+    <section class="flex-row">
+        <section class="flex-column">
+            <div class="item-title block-component">
+                <span class="item-number">1.2.3.4.5</span>
+                <span class="name">Some Thing</span>
+            </div>
+        </section>
+    </section>
+    <footer class="small block-component status inprogress">In Progress</footer>
```

Comment:
I think i'm going to add a section for the different statuses which just has the status block